tests/validator-clone/tests/validator-clone.ts

```diff
@@ -0,0 +1,66 @@
+import * as anchor from "@project-serum/anchor";
+import { Program } from "@project-serum/anchor";
+import assert from "assert";
+import { ValidatorClone } from "../target/types/validator_clone";
+
+describe("validator-clone", () => {
+  // Configure the client to use the local cluster.
+  anchor.setProvider(anchor.Provider.env());
+
+  const program = anchor.workspace.ValidatorClone as Program<ValidatorClone>;
+  const connection = program.provider.connection;
+
+  it("Clone non-executable account", async () => {
+    // Metadata program upgrade authority
+    const account = "AqH29mZfQFgRpfwaPoTMWSKJ5kqauoc1FwVBRksZyQrt";
+    const [accountInfo] = await anchor.utils.rpc.getMultipleAccounts(
+      connection,
+      [new anchor.web3.PublicKey(account)]
+    );
+    assert.ok(accountInfo !== null, "Account " + account + " not found");
+  });
+
+  it("Clone bpf2-program account", async () => {
+    // Memo program
+    const account = "MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr";
+    const [accountInfo] = await anchor.utils.rpc.getMultipleAccounts(
+      connection,
+      [new anchor.web3.PublicKey(account)]
+    );
+    assert.ok(accountInfo !== null, "Account " + account + " not found");
+  });
+
+  it("Clone bpf3-program account and its program data", async () => {
+    const accounts = [
+      // Metadata program
+      "metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s",
+      // Metadata program executable data
+      "PwDiXFxQsGra4sFFTT8r1QWRMd4vfumiWC1jfWNfdYT",
+    ];
```

Comment:
pls add one more cloned bpf3 account here (so two addresses) and in the Anchor.toml to check that the cloning also works for multiple bpf3 accounts